nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/TestTreeCommentNotification.java

```diff
@@ -31,4 +43,39 @@
     protected Class<? extends CommentManager> getType() {
         return TreeCommentManager.class;
     }
+
+    @Test
+    @Deploy("org.nuxeo.ecm.platform.comment:OSGI-INF/notification-subscription-contrib.xml")
+    public void testAutoSubscribingOnlyOnceToNewComments() {
```

Comment:
same interrogation about the use of this new `notification-subscription-contrib.xml`

Reply:
https://github.com/nuxeo/nuxeo/pull/3663#discussion_r371331461